tests/test_object_positions.py

```diff
@@ -12,7 +12,7 @@ def run_imsim(camera):
     imsim_dir = os.path.dirname(os.path.abspath(str(Path(__file__).parent)))
     os.environ['SIMS_SED_LIBRARY_DIR'] \
         = os.path.join(imsim_dir, 'tests', 'data', 'test_sed_library')
-    template = os.path.join(imsim_dir, 'config', 'imsim-config.yaml')
+    template = os.path.join(imsim_dir, 'config', 'imsim-config-instcat.yaml')
```

Comment:
You should have some kind of unit test that uses the skycat config file.  Right now, the only test is that it doesn't crash.

Reply:
I've made a new issue #397 to address this comment and adding tests for the different input cases more generally.

The change in this PR was just to make the existing test work as before as we no longer call the bare imsim-config yaml.

Reply:
BTW: Note I made a similar change in test_header_keywords.   Those were the two tests explicitly reading templates.